poetry/1.8/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM to-be-replaced-by-local-ref/base:ubi-9.4-minimal
+
+RUN set -euExo pipefail && shopt -s inherit_errexit && \
+    dnf install -y make python3.9 git && \
```

Comment:
iirc dnf is not available in ubi-minimal, you should use microdnf. CI will validate but it's going to take a while.

Reply:
right, fixed, thanks